lsp-server/src/main/scala/org/alephium/ralph/lsp/server/RalphLangServer.scala

```diff
@@ -59,9 +60,13 @@ object RalphLangServer {
   def getRootUri(params: InitializeParams): Option[URI] =
     Option(params.getRootUri)
       .orElse(Option(params.getRootPath))
+      .map(URI.create)
       //Some LSP clients aren't providing `rootUri` or `rootPath`, like in nvim, so we fall back on `user.dir`
-      .orElse(Option(System.getProperty("user.dir")).map(dir => s"file://$dir"))
-      .map(new URI(_))
+      .orElse {
+        Option(System.getProperty("user.dir"))
+          .map(Paths.get(_))
+          .map(_.toUri)
+      }
```

Comment:
@tdroxler just reaching out for a quick double check: Does the vim case mentioned in your comment here still work ok?